gravitee-am-repository/gravitee-am-repository-jdbc/src/main/java/io/gravitee/am/repository/jdbc/oauth2/api/JdbcAccessTokenRepository.java

```diff
@@ -151,15 +152,15 @@ public Completable deleteByUserId(String userId) {
                 .matching(Query.query(where(SUBJECT).is(userId)))
                 .all())
                 .doOnError(error -> LOGGER.error("Unable to delete access tokens with subject {}",
-                userId, error));
+                        userId, error));
     }
 
     @Override
-    public Completable deleteByDomainIdClientIdAndUserId(String domainId, String clientId, String userId) {
+    public Completable deleteByDomainIdClientIdAndUserId(String domainId, String clientId, UserId userId) {
         LOGGER.debug("deleteByDomainIdClientIdAndUserId({},{},{})", domainId, clientId, userId);
         return monoToCompletable(getTemplate().delete(JdbcAccessToken.class)
                 .matching(Query.query(
-                        where(SUBJECT).is(userId)
+                        where(SUBJECT).is(userId.lookupSubject())
```

Comment:
**question/issue**: I'm supprised that you have to do this lookUp as the tokens (access or refresh are supposed to have the userId as reference) :thinking: